packages/pnpm/test/run.ts

```diff
@@ -18,3 +18,16 @@ test('pnpm run: returns correct exit code', async (t: tape.Test) => {
   t.equal(execPnpmSync('run', 'exit0').status, 0)
   t.equal(execPnpmSync('run', 'exit1').status, 1)
 })
+
+
+test('pass the args to the command that is specfied in the build script', async t => {
+  const project = prepare(t, {
+    scripts: {
+      test: 'ts-node test'
+    },
+  })
+
+  const result = execPnpmSync('run', 'test', '--', '--flag=true')
+
+  t.ok((result.stdout as Buffer).toString('utf8').match(/ts-node test "--flag=true"/), 'command was successfull')
```

Comment:
Typo. It should be "successful" with a single "l" at the end
you can even omit the message in this case as it is the only assertion in the test case, which has a description